interface/patient_tracker/patient_tracker.php

```diff
@@ -65,6 +65,9 @@
 $appointments = fetch_Patient_Tracker_Events($from_date, $to_date, $provider, $facility, $form_apptstatus, $form_apptcat);
 $appointments = sortAppointments( $appointments, 'time' );
 
+//Count of arrived patient
+$arrived_status = getArrivedStatus($appointments);
```

Comment:
Wouldn't it be better to keep counts by status in an array? That will offer ability to customize display for other needs.  e.g.
`$astat[$appointment['status']] += 1;`